.github/workflows/ci_python.yml

```diff
@@ -43,6 +46,7 @@ jobs:
   #
   # Also, only do it on one machine.
   lint:
+    if: github.event_name == 'pull_request' || github.event_name == 'merge_group'
```

Comment:
Only lint on pull requests and merge queue.

Reply:
linting on commit was helpful for me today... maybe don't add this?

Reply:
It will still happen on commit -- that's part of the `pull_request` event (eg., when you update the pull request it will run).